docs/prow/prow-secrets.md

```diff
@@ -0,0 +1,39 @@
+
+# Prow Secrets
+
+## Overview
+
+This document lists all types of Secrets used in the `kyma-prow` and `workload-kyma-prow` clusters, where all the ProwJobs are executed.
+>**NOTE:** All Secrets are stored in the Google Cloud Storage (GCS) bucket.
+
+
+## kyma-prow cluster
+
+| Prow Secret   | Description | 
+| :---------- | :---------------- | 
+| **hmac-token**| Used for validating GitHub webhooks. It is manually generated using the `openssl rand -hex 20` command.| 
+| **oauth-token**| Personal access token called `prow-production` used by the `kyma-bot` GitHub user. | 
+|**sap-slack-bot-token**| Token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot). |
+|**workload-cluster**| Stores the `workload-kyma-prow` cluster certificate. |
+| **slack-token** | OAuth token for the Slack bot user. It is used by Crier. |
+
+## workload-kyma-prow cluster
+
+| Secret   | Description | 
+| :---------- | :---------------- | 
+| **whitesource** keys | Copied directly from the bucket when executing a job. These Secrets are not stored on the cluster. | 
+| **github-integration** | Used to authorize GitHub applications configured in the `kyma-project` organization. See the **[OAuth Apps](https://developer.github.com/apps/building-oauth-apps/)** section in GitHub.|
+| **sa-*** | Service Accounts used in pipelines. Find more information [here](/docs/prow/authorization.md).| 
+| **kyma-website-bot-zenhub-token** | ZenHub token for the `kyma-website-bot` account.| 
+| **kyma-bot-github-token**| Personal access token called `prow-job` used by the `kyma-bot` GitHub user.| 
+| **kyma-guard-bot-github-token** | Personal access token for the `kyma-guard-bot` GitHub account.| 
```

Comment:
This one is used by jobguard, we can change description to "Personal access token for the `kyma-guard-bot` GitHub account. It is used by jobguard."